datastream-directory/src/main/java/com/linkedin/datastream/server/DirectoryTransportProviderAdmin.java

```diff
@@ -39,6 +40,10 @@ public TransportProvider assignTransportProvider(DatastreamTask task) {
   public void unassignTransportProvider(DatastreamTask task) {
   }
 
+  @Override
+  public void unassignTransportProvider(List<DatastreamTask> taskList) {
```

Comment:
nit: Would a Set rather than a List be better to use here?

Reply:
Keeping aligned with the other interfaces used in connector that pass bunch of task object, uses list and not set.